src/SFA.DAS.QnA.Database/projects/epaoall/sections/section5.json

```diff
@@ -410,49 +410,50 @@
 			"LinkTitle": "Supporting current learners",
 			"InfoText": "",
 			"Questions": [
-				{
-					"QuestionId": "WR-03",
-					"QuestionTag": null,
-					"Label": "How will you support the learners you're not going to assess?",
-					"ShortLabel": "",
-					"InputClasses": null,
-					"QuestionBodyText": "<p class='govuk-body'>Your answer should include:</p><ul class='govuk-list govuk-list--bullet'><li>how you'll transfer these learners to other EPAOs</li><li>details of any EPAOs you've spoken to</li></ul>",
-					"Hint": "",
-					"Input": {
-						"Type": "Textarea",
-						"InputClasses": "govuk-!-width-full",
-						"Options": [
-							{
-								"FurtherQuestions": [],
-								"Value": "Yes",
-								"Label": "Yes",
-								"HintText": ""
-							},
-							{
-								"FurtherQuestions": [],
-								"Value": "No",
-								"Label": "No",
-								"HintText": ""
-							}
-						],
-						"Validations": [
-							{
-								"Name": "Required",
-								"Value": "",
-								"ErrorMessage": "Please enter how you will support learners"
-							},
-							{
-								"Name": "MaxWordCount",
-								"Value": "500",
-								"ErrorMessage": "How you support learners  must be 500 words or less"
-							}
-						],
-						"DataEndpoint": null
-					},
-					"Order": null,
-					"Value": "",
-					"ErrorMessages": null
-				}
+        {
+          "QuestionId": "WR-03",
+          "QuestionTag": null,
+          "Label": "How will you support the learners you are not going to assess?",
```

Comment:
We also making changes to the "Title" for consistency even though this isn't displayed currently with only a single question on the page

"Title": "Supporting current learners"
should be

"Title": "How will you support the learners you are not going to assess?"

For both pages